mirrord/agent/iptables/src/lib.rs

```diff
@@ -79,6 +51,8 @@ pub trait IPTables {
     fn add_rule(&self, chain: &str, rule: &str) -> IPTablesResult<()>;
     fn insert_rule(&self, chain: &str, rule: &str, index: i32) -> IPTablesResult<()>;
     fn list_rules(&self, chain: &str) -> IPTablesResult<Vec<String>>;
+
+    fn list_table(&self, table: &str) -> IPTablesResult<Vec<String>>;
```

Comment:
```suggestion
    fn list_table(&self) -> IPTablesResult<Vec<String>>;
```
This trait kind of assumes that all operations are done on the same table. See that its implementor holds the table name in a field